src/webui/webapplication.cpp

```diff
@@ -367,6 +367,8 @@ void WebApplication::action_command_download()
     CHECK_URI(0);
     QString urls = request().posts["urls"];
     QStringList list = urls.split('\n');
+    bool skipChecking = request().posts["skip_checking"].trimmed() == "true";
+    bool addPaused = request().posts["paused"].trimmed() == "true";
```

Comment:
@glassez 
Done.

`RequestParser::splitMultipartData()` would now drop extra trailing newlines.

This affects both `request().posts` and `request().files`.